components/authz-service/server/rules_property_test.go

```diff
@@ -815,7 +816,7 @@ func reportRules(t *testing.T, rules []api.CreateRuleReq) {
 	valueBytes := 0
 	for _, rule := range rules {
 		condCount += len(rule.Conditions)
-		ruleBytes += rule.XXX_Size()
```

Comment:
the `XXX_foo` methods are gone, though we should not have been using them anyway